templates/zerver/app/navbar.html

```diff
@@ -12,35 +12,37 @@
                         <span id="streamlist-toggle-unreadcount">0</span>
                     </a>
                 </div>
-                {% if search_pills_enabled %}
-                <div id="searchbox">
-                    <div id="tab_bar" class="notdisplayed">
-                    </div>
-                    <a class="search_icon" href="#search-operators" data-overlay-trigger="search-operators" title="{{ _('Search help') }}"><i class="fa fa-search" aria-hidden="true"></i></a>
-                    <form id="searchbox_form" class="form-search navbar-search">
-                        <div id="search_arrows" class="pill-container input-append">
-                            <div contenteditable="true" class="input search-query input-block-level" id="search_query" type="text" placeholder="{{ _('Search') }}"
-                              autocomplete="off" aria-label="{{ _('Search') }}" title="{{ _('Search') }} (/)"> </div>
+                <div class="top-navbar-border">
```

Comment:
Does this commit make any changes other than adding a div wrapper here?  Either way, you should answer that question clearly in the commit message -- that's the most important question anyone trying to review this will have since it impacts how they try to read this (large) potion of the diff.